build.gradle

```diff
@@ -45,11 +45,28 @@ ext {
     avro_version = "1.8.2"
 }
 
+spotless {
+    java {
+        enforceCheck false
```

Comment:
By default, `./gradlew build` will fail if the code style is violated.  Setting the flag to false suppresses that, so we can have a standalone build step in jenkins just for the code style.  This is what this branch is doing now, where I replaced the checkstyle stage in the Jenkinsfile with a spotless stage.